tests/generation/SettingsTest.py

```diff
@@ -16,6 +16,13 @@
 
 class SettingsTest(unittest.TestCase):
 
+    def path_leaf(self, path):
+        """
+        :return: The file name from the given path
```

Comment:
Missing full stop.

Reply:
Missing full stop in commit messages as well

Reply:
Also, you can convert those three lines into pts
```
* line 1
* line 2
* line 3
```
Looks much cleaner and organized.